src/jit/jiteh.cpp

```diff
@@ -637,6 +660,59 @@ bool            Compiler::ehIsBlockEHLast(BasicBlock* block)
            (ehIsBlockHndLast(block) != nullptr);
 }
 
+/******************************************************************************************
+* Return the EH descriptor for the most nested region that may handle exceptions raised in this BasicBlock
+* (or nullptr if this block's exceptions propagate to caller).
+*/
+EHblkDsc*       Compiler::ehGetBlockExnFlowDsc(BasicBlock* block)
```

Comment:
coding conventions: 4-space indents. Please use the new style function header comment (see doc on GH, or conventions.txt in JIT directory)